akka-docs/rst/scala/http/server-side-https-support.rst

```diff
@@ -21,6 +21,9 @@ For detailed documentation for client-side HTTPS support refer to :ref:`clientSi
 
 .. _akka.http.scaladsl.Http: https://github.com/akka/akka/blob/master/akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala
 
+
+.. _ssl-config:
```

Comment:
see above `-scala` and `-java` suffixes needed, otherwise `:ref:` links will break with ambiguous targets